packages/safe-fetch/src/lib/fetch.ts

```diff
@@ -0,0 +1,50 @@
+import { err, ok, type Result } from '@sapphire/result';
```

Comment:
Add triple slash directive (see @sapphire/fetch).

This is important for users as they don't have the tsconfig this folder has.

Reply:
Done in 891664e